internal/mode/static/handler.go

```diff
@@ -29,23 +29,13 @@ import (
 	"github.com/nginxinc/nginx-gateway-fabric/internal/mode/static/state/graph"
 	"github.com/nginxinc/nginx-gateway-fabric/internal/mode/static/state/resolver"
 	"github.com/nginxinc/nginx-gateway-fabric/internal/mode/static/status"
+	"github.com/nginxinc/nginx-gateway-fabric/internal/mode/static/telemetry"
 )
 
 type handlerMetricsCollector interface {
 	ObserveLastEventBatchProcessTime(time.Duration)
 }
 
-//go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 -generate
-//counterfeiter:generate . secretStorer
-
-// secretStorer should store the usage Secret that contains the credentials for NGINX Plus usage reporting.
-type secretStorer interface {
-	// Set stores the updated Secret.
-	Set(*v1.Secret)
-	// Delete nullifies the Secret value.
-	Delete()
-}
-
 // eventHandlerConfig holds configuration parameters for eventHandlerImpl.
```

Comment:
nit: mixed usage of `config` and `configuration` in the file.

Reply:
This is probably a pretty common pattern throughout the code, I don't think it's necessarily worth addressing.